src/adaptors/adaptor_common.c

```diff
@@ -72,28 +78,110 @@ qd_error_t qd_load_adaptor_config(qd_adaptor_config_t *config, qd_entity_t *enti
     return qd_error_code();
 }
 
+void qd_adaptor_common_init(void)
+{
+    if (buffer_ceiling > 0) {
+        return;
+    }
+
+    char     *ceiling_string = getenv("SKUPPER_MEMORY_CEILING");
```

Comment:
Should this be called `SKUPPER_ROUTER_MEMORY_CEILING` instead of `SKUPPER_MEMORY_CEILING`

Reply:
Yes, probably.